users/utils.py

```diff
@@ -0,0 +1,35 @@
+import re, jwt
+
+from functools import wraps
+
+from django.http import JsonResponse
+
+from users.models import User
+from my_settings  import SECRET_KEY, ALGORITHM
+
+def email_validation(email):
+    EMAIL_REGEX = re.match(r"^(\w+[+-_.]?\w?)+@([\w+-_.]+[.][\w+-_.]+)$", email)
+    return EMAIL_REGEX
+
+def password_validation(password):
+    PASSWORD_REGEX = re.match(r"^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&?*])([a-zA-Z0-9!@#$%^&?*]){8,}$", password)
+    return PASSWORD_REGEX
```

Comment:
데코레이터 용도로 사용하는 함수가 아니라면 함수명은 명확한 의미 파악을 위해 동사로 시작해주시는게 좋습니다.
```suggestion
def validate_email(email):
    EMAIL_REGEX = re.match(r"^(\w+[+-_.]?\w?)+@([\w+-_.]+[.][\w+-_.]+)$", email)
    return EMAIL_REGEX

def validate_password(password):
    PASSWORD_REGEX = re.match(r"^(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*[!@#$%^&?*])([a-zA-Z0-9!@#$%^&?*]){8,}$", password)
    return PASSWORD_REGEX
```

Reply:
넵 수정하였습니다!